scripts/deploy-ebpf.ps1.in

```diff
@@ -48,6 +48,10 @@ $source_directory="."
     "bindmonitor_ringbuf_um.dll",
     "bindmonitor_ringbuf_um.pdb",
     "bindmonitor_ringbuf.sys",
+    "bindmonitor_mt_tailcall.o",
```

Comment:
what about updating `installer\Product.wxs`?
(@gtrevi to confirm, all test files should also be listed in there, correct?)